### PR TITLE
Added 2 updates

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
1) no recycle bin limitation
2) clarity that the Files Tab just needs to be refreshed once after the rename.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/change-site-address.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ Flows will need to be recreated after the site address change.
 The Word, Excel, and PowerPoint desktop apps and apps for the web will show the new URL after the change.
 
 **Recycle bin**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:143]:
```suggestion
**Recycle Bin**<br>
```

File: SharePoint/SharePointOnline/change-site-address.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ Flows will need to be recreated after the site address change.
...
-Any files in the recycle bin will be lost when the site address is changed.
+Files in the Recycle Bin will be restorable as per usual deletion timeframe.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:144]:
```suggestion
Files in the Recycle Bin will be restorable as per the usual deletion timeframe.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**